test/python/test_cudnn_sdpa_op.py

```
@@ -143,11 +149,15 @@ def sdpa_reference_fwd_bwd(
 def _skip_if_unsupported_d256(D):
     if D != 256:
         return
+    if cudnn.backend_version() >= _CUDNN_BACKEND_D256_VERSION:
+        return
     major, minor = torch.cuda.get_device_capability()
```

Ruff (0.15.15)

[warning] Unpacked variable `minor` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)